app/knowledge_panels.py

```diff
@@ -16,14 +18,25 @@ def hunger_game_kp(
         country = value
         hunger_game_filter = value = None
     if country is not None:
-        query["country"] = country
+        query["country"] = f"en:{country}"
         description = country
     if hunger_game_filter is not None:
-        query["type"] = f"{hunger_game_filter}"
-        description = f"{hunger_game_filter}"
-    if value is not None:
-        query["value_tag"] = value
-        description = f"{value} {hunger_game_filter}"
+        if hunger_game_filter == "brand":
+            query["brand"] = value
+            description = f"brand {value}"
+            if sec_value is not None:
+                query["type"] = f"{sec_facet}"
+                query["value_tag"] = sec_value
+                description += f" {sec_facet} {sec_value}"
+        else:
+            query["type"] = f"{hunger_game_filter}"
+            description = f"{hunger_game_filter}"
+            if value is not None:
+                query["value_tag"] = value
+                description = f"{hunger_game_filter} {value}"
+            if sec_facet == "brand":
+                query[sec_facet] = sec_value
+                description += f" {sec_facet} {sec_value}"
```

Comment:
This is quite close, but it's a bit messy.

Once again I will say, we have to unite and process.

Moreover I would expect that in the case we have two facets that are not mergable into one hunger game query, we return a double knowledge panel, with question for one facets and for the other.

If you are ok to try, I would say:
1. first unite all facets in a dict `facets = {hunger_game_filter: value, sec_facet: sec_value}`
2. eventually remove [None] keys and [None] values (it does not have sense in hungergame to filter without a value)
3. if there is a "brand" key, use it as a global filter and pop it out from the dict
4. for each key, value remaining in the dict build the question url, eventually adding brand filter, if there is one
5 build the knowledge panel

Of course you will have to change tests and try different test cases.

Reply:
oh sorry, you're right, keep them :-)
I let you handle the logic in the above schema.